subprojects/base-services/src/test/groovy/org/gradle/internal/operations/BuildOperationIdentifierRegistryTest.groovy

```diff
@@ -0,0 +1,21 @@
+package org.gradle.internal.operations
+
+import spock.lang.Specification
+import spock.lang.Subject
+
+@Subject(BuildOperationIdentifierRegistry)
```

Comment:
Yeah, as with other tests I did this. I'm sure the IDE finds it anyway, but this makes it _super_ clear. I don't think we have a convention here.